build/karma.conf.js

```diff
@@ -92,7 +83,6 @@ module.exports = config =>
         files: [
             'https://cdn01.boxcdn.net/polyfills/core-js/2.5.3/core.min.js',
             `src/third-party/model3d/${MODEL3D_STATIC_ASSETS_VERSION}/three.min.js`,
-            `src/third-party/doc/${DOC_STATIC_ASSETS_VERSION}/**/*.js`,
```

Comment:
Intentional?

Reply:
Yep, intentional! PhantomJS chokes on the latest pdf.js assets and they aren't actually necessary, anyway. We only have one test that depended on them, so I just moved it over to a simple mock object.